drivers/sensor/icm42688/Kconfig

```diff
@@ -42,6 +49,14 @@ config ICM42688_TRIGGER_OWN_THREAD
 
 endchoice
 
+config ICM42688_STREAM
+	bool "Use hardware FIFO to stream data"
+	select ICM42688_TRIGGER
+	default y
+	depends on SPI_RTIO
+	help
+	  Use this config option to enable streaming sensor data via RTIO subsystem.
+
```

Comment:
You should just use `SENSOR_ASYNC_API` to control the API. If the dev enabled it, they would expect it to work on all the sensors.